### DISCUSSION TITLE
Disable extra `lang.typescript` from LazyExtras stale issue

### DISCUSSION THREAD

**Discussion by tuephan-hi** [LazyVim/LazyVim#3390] [General]
Hey folks,

I'm turning on the `lang.typescript` in the extra to try out the vtsls LSP but when turning off, this LSP still loads whenever I open the typescript file. I had to do a complete wipe (commands below, maybe overkill) to actually removing the LSP from the config.
Seems like there is staleness issue in the config whenever we disable stuffs from the Extras. It would be good to address this. Thanks.

```
rm -rf ~/.local/share/nvim/
rm -rf ~/.local/state/nvim/
rm -rf ~/.cache/nvim
```

**Comment by dpetka2001** [comment]
LazyVim uses `Mason` to install the LSP servers, but Mason only supports automatic installation. That means the user has to manually uninstall the LSP servers from Mason UI himself.
